### DISCUSSION TITLE
One piece

### DISCUSSION THREAD

**Discussion by DomDory** [aleganza/akuse#150] [General]
When watching one piece on akuse it has a hard time loading episodes and always saves on an episode i already watched and it only is a problem when i watch one piece 
![Screenshot 2024-09-07 161329](https://github.com/user-attachments/assets/18ac03e1-5ad8-4025-98a2-4afc8ae3e699)

**Comment by aleganza** [comment]
I'm gonna add a light mode where it doesn't load all the data
